app/src/main/java/de/westnordost/streetcomplete/quests/address/AddHousenumber.kt

```diff
@@ -19,7 +19,7 @@ import de.westnordost.streetcomplete.util.isInMultipolygon
 
 class AddHousenumber(private val overpass: OverpassMapDataAndGeometryDao) : OsmElementQuestType<HousenumberAnswer> {
 
-    override val commitMessage = "Add housenumbers"
+    override val commitMessage = "Add address"
```

Comment:
```suggestion
    override val commitMessage = "Add housenumbers"
```

I think it should keep describing exactly what had been added.